lib/glific/contacts/import.ex

```diff
@@ -12,22 +12,64 @@ defmodule Glific.Contacts.Import do
       phone: data["phone"],
       organization_id: organization_id,
       language_id: Enum.at(Settings.get_language_by_label_or_locale(data["language"]), 0).id,
-      optin_time: elem(Timex.parse(data["opt_in"], "{YYYY}-{0M}-{0D}"), 1)
```

Comment:
can we make this a bit more flexible, and maybe take it as an option with default value

Reply:
Yup got this!